include/OpenColorIO/OpenColorIO.h

```diff
@@ -2825,10 +2825,12 @@ class OCIOEXPORT GpuShaderCreator
                             const Float3Getter & getFloat3) = 0;
 
     virtual bool addUniform(const char * name,
+                            const size_t & maxSize,
```

Comment:
@Morteeza The public API cannot be changed like that because the MSL translation is planned for a patch version. 
So, that change is a blocker for OCIO 2.1.1 !

Reply:
@hodoulp got it. I will change this to not touch the public API.